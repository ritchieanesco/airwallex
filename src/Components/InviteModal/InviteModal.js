import React from "react";
import { withStyles } from "@material-ui/core/styles";
import Modal from "@material-ui/core/Modal";
import { object, node, bool, func } from "prop-types";

const styles = theme => ({
  root: {
    flexGrow: 1
  },
  paper: {
    position: "absolute",
    width: "280px",
    backgroundColor: theme.palette.background.paper,
    boxShadow: theme.shadows[5],
    padding: theme.spacing.unit * 4,
    top: "50%",
    left: "50%",
    transform: "translate(-50%, -50%)"
  }
});

const InviteModal = props => {
  const { classes } = props;
  return (
    <Modal
      aria-labelledby="invite-modal-title"
      aria-describedby="invite-modal-description"
      open={props.open}
      onClose={props.onClose}
    >
      <div className={classes.paper}>{props.children}</div>
    </Modal>
  );
};

InviteModal.propTypes = {
  classes: object,
  children: node,
  open: bool,
  onClose: func
};

export default withStyles(styles)(InviteModal);
