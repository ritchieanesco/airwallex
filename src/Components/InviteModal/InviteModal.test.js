import React from "react";
import renderer from "react-test-renderer";
import InviteModal from "./InviteModal";

describe("InviteModal", () => {
  it("should render correctly", () => {
    const tree = renderer.create(<InviteModal open={false} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
