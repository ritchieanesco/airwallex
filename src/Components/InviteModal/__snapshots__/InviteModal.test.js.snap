// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InviteModal should render correctly 1`] = `null`;
